src/views/Upload.vue

```diff
@@ -69,6 +88,7 @@ export default {
 			tabs: ['Nahrať súbory', 'Vložiť text'],
 			files: [],
 			text: '',
+			checked: false,
```

Comment:
What if we used variable with meaningful name. Maybe `userEmailProvided` or something like that?